packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -208,15 +208,16 @@ export class CommandBarBase extends BaseComponent<ICommandBarProps, {}> implemen
     return <OverflowButtonType aria-posinset={ this._setSize } aria-setsize={ this._setSize } { ...overflowProps as IButtonProps } />;
   }
 
-  private _computeCacheKey(primaryItems: ICommandBarItemProps[], farItems: ICommandBarItemProps[], overflow: boolean): string {
+  private _computeCacheKey(data: ICommandBarData): string {
+    const { primaryItems, farItems = [], overflowItems } = data;
     const returnKey = (acc: string, current: ICommandBarItemProps): string => {
       const { cacheKey = current.key } = current;
       return acc + cacheKey;
     };
 
     const primaryKey = primaryItems.reduce(returnKey, '');
     const farKey = farItems.reduce(returnKey, '');
-    const overflowKey = overflow ? 'overflow' : '';
+    const overflowKey = !!overflowItems.length ? 'overflow' : '';
```

Comment:
Is there a concern if `overflowItems` is undefined? It's listed as an optional property.

Reply:
The data that is kicked around inside has a different interface

![image](https://user-images.githubusercontent.com/1434956/40395558-869bf4e6-5ddd-11e8-8dda-958f2b16863e.png)

We let farItems be undefined so that !farItems means to omit the far items markup. We could set farItems to an empty array and test by length but eh